src/store/reducer.tsx

```diff
@@ -0,0 +1,64 @@
+import { TEST_ACTION } from "./actions";
+import { createSlice } from "@reduxjs/toolkit";
+
+interface TestReducerState {
+  posts: UserObject[];
+  idCount: number;
+}
+
+export interface UserObject {
+  userId: number;
+  id: number;
+  title: string;
+  body: string;
+}
+
+const initialState: TestReducerState = {
+  posts: [],
+  idCount: 0,
+};
+
+export const testSlice = createSlice({
+  name: "testSlice",
+  initialState: initialState,
+  reducers: {
+    addPost(state) {
+      state.posts.push({
+        id: state.idCount,
+        userId: state.idCount,
+        body: "heyhey",
+        title: "hey",
+      });
+      state.idCount += 1;
+    },
+  },
+});
+
+export default testSlice.reducer;
+export const { addPost } = testSlice.actions;
+
+// const testReducer = (
```

Comment:
it's better to remove commented out code. especially if you already replaced it with new implementation